### PR TITLE
[DPE-1935][DPE-1754] Postgresql logs

### PR DESCRIPTION
**Pull request by dragomirp**
## Issue
Postgresql's logs are not split into a separate location

## Solution
Split the Postgresql logs and add the log rotation configuration from the spec.

### CODE REVIEW

File: src/grafana_dashboards/postgresql-metrics.json [hunk 1]
```diff
@@ -3033,7 +3033,7 @@
   "style": "dark",
   "tags": [
     "postgres",
-    "db",
```
Comment (dragomirp) [inline src/grafana_dashboards/postgresql-metrics.json:3036]: Typo

File: templates/patroni.yml.j2
Comment (dragomirp) [inline templates/patroni.yml.j2:1]: Copy of the configuration in the spec.

Comment (dragomirp) [inline templates/patroni.yml.j2:1]: Mostly copied from the spec.

**Review by taurus-forever — APPROVED** [review]
Tnx!

**Pull request merged**